packages/kbn-interpreter/src/public/interpreter.test.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { initializeInterpreter } from './interpreter';
+
+jest.mock('../common/interpreter/interpreter_provider', () => ({
+  interpreterProvider: () => () => ({}),
+}));
+
+jest.mock('../common/lib/serialize', () => ({
+  serializeProvider: () => ({ serialize: () => ({}) }),
+}));
+
+jest.mock('./create_handlers', () => ({
+  createHandlers: () => ({}),
+}));
+
+describe('kbn-interpreter/interpreter', () => {
+  it('loads server-side functions', async () => {
+    const kfetch = jest.fn(async () => ({}));
+
+    await initializeInterpreter(kfetch, { toJS: () => ({}) }, ({ register: () => {} }));
+
+    expect(kfetch).toHaveBeenCalledTimes(1);
+    expect(kfetch).toHaveBeenCalledWith({ pathname: '/api/canvas/fns' });
```

Comment:
nit: pulling the path prefix from the constants file means that changing the path won't break tests... and I suspect we're going to change it since this isn't just for canvas anymore.

Reply:
Makes sense.